TobiiTest/Preference.xaml.cs

```diff
@@ -53,6 +53,11 @@ public Preference(Preferences pr)
             ssSizeCB.SelectionChanged += SsSizeCB_SelectionChanged;
             sizeXTB.TextChanged += SizeXTB_TextChanged;
             sizeYTB.TextChanged += SizeYTB_TextChanged;
+
+            setKey.SelectedItem = setKey.Items.IndexOf(prefs.Get("key"));

```

Comment:
`SelectedItem` is an object in the ComboBox. And you try to assign number (`IndexOf`) to it.
```suggestion
            setKey.SelectedIndex = setKey.Items.IndexOf(prefs.Get("key"));
```